seed-gen/src/buttercup/seed_gen/vuln_base_task.py

```diff
@@ -256,13 +256,12 @@ def submit_valid_pov(
             ctoken,
         )
 
-        crash = Crash(
-            target=build,
-            harness_name=self.harness_name,
-            crash_input_path=dst,
-            stacktrace=stacktrace,
-            crash_token=ctoken,
-        )
+        crash = Crash()
+        crash.target.CopyFrom(build)
+        crash.harness_name = self.harness_name
+        crash.crash_input_path = dst
+        crash.stacktrace = stacktrace
+        crash.crash_token = ctoken
```

Comment:
Why the change?

Reply:
It seems to be related to the new protobuf version :/ I couldn't assign things like before anymore as it was complaining at runtime.